utility/utility.go

```diff
@@ -36,6 +36,7 @@ const (
 	CatchupPath      = "catchup_" + VersionStr + "/"
 	WalPath          = "wal_" + VersionStr + "/"
 	BackupNamePrefix = "base_"
+	BackupTimeFormat = "20060102T150405Z"
```

Comment:
Just wonder if this variation of iso 8601 have a specific name...
but who cares?